hugo/assets/scss/base/base.scss

```diff
@@ -12,18 +13,14 @@
     --text-color: #{ $c-grey--darkest };
     --heading-color: #{ $c-blue };
     --text-link-color: #{ $c-blue };
-    --caption-color: #{ transparentize($c-grey--darkest, .3) };
+    --caption-color: #{ transparentize($c-grey--darkest, 0.3) };
     --list-color: #{ $c-blue };
-    --table-border-color: #{ transparentize($c-blue--light, .85) };
-    --table-header-border-color: #{ transparentize($c-blue--light, .85) };
+    --table-border-color: #{ transparentize($c-blue--light, 0.85) };
+    --table-header-border-color: #{ transparentize($c-blue--light, 0.85) };
     --pre-color: #{ $c-grey--darkest };
     --pre-background-color: #{ $c-blue--lighter };
     --pre-border-color: #{ $c-blue--lighter };
-    --quote-background-color: #{ $c-grey--lighter };
```

Comment:
why are these removed here? for base stuff it's best to set all variables at the top so it's easily to change them all if you have to (for instance different styling needed when base stuff is shown on a blue background)